<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fenetre_generation_clef_mdp</class>
 <widget class="QWidget" name="Fenetre_generation_clef_mdp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>611</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="global">
    <item row="7" column="3">
     <widget class="QPushButton" name="bouton_copier">
      <property name="text">
       <string>Copier</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Taille à générer :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>                     </string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="4">
     <widget class="QTextEdit" name="resultat"/>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>             </string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="bouton_generer">
      <property name="text">
       <string>Générer</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLabel" name="error">
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QLineEdit" name="edit_taille">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Résultat:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
